packages/@react-spectrum/menu/stories/MenuTrigger.stories.tsx

```diff
@@ -206,27 +501,41 @@ storiesOf('MenuTrigger', module)
           <input placeholder="Tab here" />
         </div>
       </>
-    )
+    ),
+    {chromatic: {disable: true}}
   );
 
-function render({isDisabled, ...props}: any = {}, menuProps = {}) {
+let customMenuItem = (item) => {
+  let Icon = iconMap[item.icon];
+  return (
+    <Item childItems={item.children} textValue={item.name} key={item.name}>
+      {item.icon && <Icon size="S" />}
+      <Text>{item.name}</Text>
+      {item.shortcut && <Keyboard>{item.shortcut}</Keyboard>}
+    </Item>
+  );	
+};
+
+function render(menu, {isDisabled, ...props}: any = {}, menuProps = {}) {
```

Comment:
Can we have two functions? render and renderCustom?
i don't like passing null to all those render functions up above :-/
i know you did it so that the custom render ones would look like render, but they're still being inserted into something

other option would be to pull the the default Menu in this function out to its own component maybe and pass that in place of null

Reply:
yeahhh, I had two functions at first but felt it was too cumbersome, I'll go and pull the default Menu out